app/src/main/java/pl/droidevs/books/about/AboutActivity.java

```diff
@@ -0,0 +1,98 @@
+package pl.droidevs.books.about;
+
+import android.arch.lifecycle.ViewModelProviders;
+import android.support.annotation.IntegerRes;
+import android.support.constraint.ConstraintLayout;
+import android.support.transition.Fade;
+import android.support.transition.TransitionInflater;
+import android.support.transition.TransitionSet;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.ImageView;
+
+import java.util.List;
+
+import pl.droidevs.books.R;
+
+public class AboutActivity extends AppCompatActivity {
+    private static final String EXTRAS_SELECTED_CONTRIBUTOR_INDEX = "EXTRAS_SELECTED_CONTRIBUTOR_INDEX";
+
+    private AboutViewModel viewModel;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setTitle(R.string.about);
+        setContentView(R.layout.activity_about);
+
+        viewModel = ViewModelProviders.of(this).get(AboutViewModel.class);
+
+        if (savedInstanceState != null) {
+            int selectedContributorIndex = savedInstanceState.getInt(EXTRAS_SELECTED_CONTRIBUTOR_INDEX, 0);
+            viewModel.selectContributor(selectedContributorIndex);
+        } else {
+            getSupportFragmentManager()
+                    .beginTransaction()
+                    .add(R.id.fl_about, AboutVPFragment.newInstance(), AboutFragment.TAG)
+                    .commit();
+        }
+    }
+
+    @Override
+    protected void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+        outState.putInt(EXTRAS_SELECTED_CONTRIBUTOR_INDEX, viewModel.getSelectedContributorIndex().getValue());
+    }
+
+    public void showContributorDetails(ImageView imageView, ConstraintLayout constraintLayout, int index) {
+
+        int FADE_DEFAULT_TIME = 250;
+        int MOVE_DEFAULT_TIME = 500;
+
+        Fragment previousFragment = getSupportFragmentManager().findFragmentById(R.id.fl_about);
+        Fragment nextFragment = ContributorFragment.newInstance();
+        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction()/*.setReorderingAllowed(true)*/;
```

Comment:
w jakim celu tu jest zostawiony zakomentowany kod?

Reply:
Bo na początku różnie animacje się zachowywały przy __setReorderingAllowed(true)__. Ale planuję to odkomentować 😉